integration-tests/injecting-multiple-manager-operations-in-a-block.spec.ts

```diff
@@ -3,7 +3,8 @@ import { CONFIGS } from "./config";
 
 CONFIGS().forEach(({ lib, rpc, setup, protocol }) => {
   const Tezos = lib;
-  const limanet = (protocol === Protocols.PtLimaPtL) ? test : test.skip;
+  const limanet = protocol === Protocols.PtLimaPtL && !(rpc === 'http://localhost:20000') ? test : test.skip;
+  const flextesaLima = protocol === Protocols.PtLimaPtL && rpc === 'http://localhost:20000' ? test : test.skip;
```

Comment:
lima and mumbaiAndAlpha could be combined right?

Reply:
mumbaiAndAlpha doesn't need the extra condition

Reply:
can make sure that its not protocol lima and localhost
 which would still allow lima and mumbai flextesa

Reply:
but what about mumbai flextesa? that would just need another condition then?